BrightCoders-Institute/BCDIC22-RoR-reloj-checador-team2
# Feature#24 store database registration

# Description

What did you implemented/Why did you implemented this:

- Store register form created
- A form with name and address inputs, and a register button 

## Test

* Move to /stores/new path or click in the add store button in the /store path
* Fill the form and submit
* Check the store's table for the store created

## Screenshots

![store_Register](https://user-images.githubusercontent.com/72171554/214439010-fce206d3-f4d6-4591-96c9-c3fcf1cc5da5.JPG)
